### ISSUE TITLE
Update project structure to use (AMD & Tests frameworks)

### ISSUE THREAD

**Issue by ahmdrefat** [tailog/tailog-ui#3]

**Comment by ikhattab** [comment]
if we are going to use https://github.com/kriasoft/react-starter-kit I believe that we are done it uses [webPack] (http://webpack.github.io/) for bundling so we can use CommonJS modules for testing react components will be tested using Jest and for e2e testing we can use protactor which is already installed.

I believe we'd have our directory structure like:

- `tests/e2e` for e2e tests
- `src/components/COMPONENT_NAME/__tests__` for testing react components

What do you think?

**Comment by ahmdrefat** [comment]
Yes, I agree. But we need to add the Protractor configs first before we start.
@ikhattab I'll push the initial structure tonight, so you would be able to add the Protractor configs, okay?

**Comment by ikhattab** [comment]
Yes!
